christina-cook/rancid-tomatillos
# Feature/form component

##Is this a fix or a feature?
This is a feature.

##What is the change/fix?
Created a form component to add search functionality. Users can now search for a movie by title.

##Where should the reviewer start?
Form.js and App.js

##How should this be tested?
Click the search icon on the main page and search for a movie to see a filtered display.

##Screenshots(if appropriate)
